starknet_devnet/starknet_wrapper.py

```diff
@@ -83,6 +83,7 @@ def __init__(self, config: DevnetConfig):
         self.starknet: Starknet = None
         self.__current_cached_state = None
         self.__initialized = False
+        self.hide_predeployed_accounts = config.hide_predeployed_accounts
```

Comment:
I would say this is not necessary because `self.config = config` is already done.